interview-questions.md

```diff
@@ -8,52 +8,56 @@ Then, research the question to expand on your answer. Even if you feel you have
 1. Rails follows an MVC pattern. What does that mean?
 
   Your answer:
-
+    MVC stands for model view controller and is the framework for how rails is structured.
   Researched answer:
-
-
+    Model View Controller is an architectural pattern that separates business logic (backend data)
+    from the input and presentation logic associated with a GUI. When a controller gets a request it
+    invokes and interacts with the model, then renders the final complete web page to the user's browser
 
 2. What is a relational database? Are there other kinds of databases?
 
   Your answer:
-
+    A relational database is where data is related to each other and organized by rows and columns. Each piece of data in the row is related to what the row represents.
   Researched answer:
-
-
+    A relational database is is a type of database that stores and provides access to data points that are related to one another. The columns of the table hold attributes of the data, and each record usually has a value for each attribute, making it easy to establish the relationships among data points.
```

Comment:
This is a good research answer. How the tables relate to one another is really important.